Sources/PIRProcessDatabase/ProcessDatabase.swift

```diff
@@ -17,7 +17,7 @@ import Foundation
 import HomomorphicEncryption
 import HomomorphicEncryptionProtobuf
 import Logging
-import PrivateInformationRetrieval
+@preconcurrency import PrivateInformationRetrieval
```

Comment:
I am wondering what needs to change in `PrivateInformationRetrieval` to allow regular import?
Did you already made those changes, but forgot to remove the `@preconcurrency`?

Reply:
Oops, missed that. I’ve now conformed `KeywordDatabaseShard` to `Sendable` and removed the `@preconcurrency`.